test/refactoring/ConvertAsync/basic.swift

```diff
@@ -391,11 +391,6 @@ func nonEscapingCompletion(completion: (Int) -> Void) { }
 func multipleResults(completion: @escaping (Result<String, Error>, Result<String, Error>) -> Void) { }
 // MULTIPLE-RESULTS: func multipleResults(completion: @escaping (Result<String, Error>, Result<String, Error>) -> Void) async { }
 
-// RUN: %refactor -add-async-alternative -dump-text -source-filename %s -pos=%(line+2):1
-// RUN: %refactor -convert-to-async -dump-text -source-filename %s -pos=%(line+1):1 | %FileCheck -check-prefix=NOT-LAST %s
-func completionNotLast(completion: @escaping (String) -> Void, a: Int) { }
```

Comment:
Again: What’s the reason for deleting this test case?

Reply:
Covered by the tests in `handler_position.swift` instead, ie. `completionFirst`